ubidots/ubidots_in.js

```diff
@@ -90,6 +89,31 @@ module.exports = function (RED) {
         }
       });
     });
+
+    function defineOutputObject(topic, message) {
+      let finalObject = {};
+      let variable = topic;
+      if (useCustomTopics) {
+        variable = topic.substring(14);
+        if (topic.substring(topic.length - 3) === "/lv") {
```

Comment:
You can simplify this by using [String.prototype.endsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith)